Unwrap/MainTabBarController.swift

```diff
@@ -22,19 +22,20 @@ class MainTabBarController: UITabBarController, Storyboarded {
         viewControllers = [home.navigationController, learn.splitViewController, practice.splitViewController, challenges.splitViewController, news.splitViewController]
     }
 
-    /// If we get some launch options, figure out which one was requested and jump right to the correct tab.
+    /// Figures out which launch option was requested (if any) and jumps right to the correct tab.
     func handle(_ launchOptions: [UIApplication.LaunchOptionsKey: Any]?) {
         if let item = launchOptions?[.shortcutItem] as? UIApplicationShortcutItem {
             handle(shortcutItem: item)
         }
     }
 
     func handle(shortcutItem: UIApplicationShortcutItem) {
-        if shortcutItem.type == "com.hackingwithswift.unwrapswift.challenges" {
+        switch shortcutItem.type {
+        case "com.hackingwithswift.unwrapswift.challenges":
```

Comment:
Could be better to use enum

Reply:
True, but that requires to create it, my idea was to optimise existing code

Reply:
Okay.